tracer/test/snapshots/AwsLambdaTests.verified.txt

```diff
@@ -240,13 +300,43 @@ at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, Cancellat
     }
   },
   {
-    TraceId: Id_1,
-    SpanId: Id_11,
+    TraceId: Id_7,
+    SpanId: Id_30,
```

Comment:
I'll take your word for it that this all looks correct 😂 AFAICT:

- All the "NoContext" spans have their own trace
- All the "WithContext" spans belong to the same TraceId (Id_7)
- All the spans have a "missing" span as their `ParentId`

All sound right to you?

Reply:
💯